pandas/tests/arrays/categorical/test_api.py

```diff
@@ -337,12 +338,31 @@ def test_remove_categories(self):
         # inplace == True
         res = cat.remove_categories("c", inplace=True)
         tm.assert_categorical_equal(cat, new)
+
+        # for removing duplicates
+        cat = Categorical(["a", "b", "c", "a"], ordered=True)
+        old = cat.copy()
+        new = Categorical([np.nan, "b", "c", np.nan], categories=["b", "c"], ordered=True)
+
+        res = cat.remove_categories("a")
+        tm.assert_categorical_equal(cat, old)
+        tm.assert_categorical_equal(res, new)
+
+        res = cat.remove_categories(["a"])
+        tm.assert_categorical_equal(cat, old)
+        tm.assert_categorical_equal(res, new)
+
```

Comment:
we're only dealing with the message in the Exception in this PR so remove these tests.